e2e/fixtures/waku-router/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "waku-router",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "waku dev",
+    "build": "waku build",
+    "start": "waku start"
```

Comment:
Is it intentional to not add `--with-ssr`?

Reply:
It's okay for here since test case will run both cases

Reply:
I see.